Sources/SwiftSyntax/Raw/RawSyntax.swift

```diff
@@ -148,7 +148,8 @@ extension RawSyntax {
   }
 
   /// Whether or not this node is a token one.
-  var isToken: Bool {
+  @_spi(RawSyntax)
+  public var isToken: Bool {
```

Comment:
As far as the type itself is marked as `@_spi(Name)`, those members doesn't need to be `@_spi(Name)`. We can just make them `public`, without `@_spi`. (I know other members e.g. `var kind` has it).